values.yaml

```diff
@@ -9,10 +9,24 @@ env: prod
 rbac:
   enabled: false
 
-# Use tls or not, if set to true, please refer to ingress.yaml and create tls secrets accordingly
-# Reference: https://kubernetes.io/docs/concepts/services-networking/ingress/#tls
-tls:
-  enabled: false
+ingress:
+  # Use tls or not, if set to true, please refer to ingress.yaml and create tls secrets accordingly
+  # Reference: https://kubernetes.io/docs/concepts/services-networking/ingress/#tls
+  tls: false
+  hosts:
+    api: test-api.screwdriver.cd
+    ui: test-cd.screwdriver.cd
+    store: test-store.screwdriver.cd
+  # Enabled singleHost if your api, store, ui are sharing the same host with different path
```

Comment:
this is not necessarily `singleHos`t, I can setup a proxy which does routing similar to this, and still send to different hosts.